app/templates/sites/index.hbs

```diff
@@ -31,6 +32,9 @@
       <th class="au-c-data-table__header-title"></th>
       <th class="au-c-data-table__header-title">Locatie</th>
       <th class="au-c-data-table__header-title">Type vestiging</th>
+      <th class="au-c-data-table__header-title">Primair telefoonnummer</th>
+      <th class="au-c-data-table__header-title">Email</th>
```

Comment:
**Nitpick:** The ticket specifies "E-mail" (with a dash) as column label instead of "Email" (which also has a different meaning in Dutch).

Reply:
https://binnenland.atlassian.net/browse/CLBV-657 Check comments on the green pill and grey pill.